scripts/uiupdate.sh

```diff
@@ -5,18 +5,15 @@ set -e
 if [ -z "$UI_VERSION_FILE" ]; then
 	echo "Must set UI_VERSION_FILE"; exit 1
 fi
-if [ -z "$UI_CLONE_DIR" ]; then
-	echo "Must set UI_CLONE_DIR"; exit 1
-fi
 
 shafileabs="$(pwd)/${UI_VERSION_FILE}"
-cd "${UI_CLONE_DIR}"
-V="$(git log -n1 --pretty=oneline)"
-echo "==> Setting UI version to: $V"
+V="$(git ls-remote https://github.com/hashicorp/boundary-ui main)"
+SHA=${V:0:40}
```

Comment:
To simplify this I would delete `SHA` transformation and leave `V` as `git ls-remote https://github.com/hashicorp/boundary-ui main | cut -f 1`. That will print only tge SHA without the reference

Reply:
Personally I like the bash substring solution, instead of needing `cut` in a subshell. However if we use that we should update the shabang to explicitly use bash, ie:

```bash
#!/usr/bin/env bash
```

Reply:
What @tmessi mentioned. I went with this less pretty solution because it doesn't need `cut`. I am fine with either direction though. Just let me know what you would prefer. I will remove the changelog edit as well.

Reply:
I do not have strong opinion at all, bash substring it is!!

Reply:
Unresolving this since the shebang line still needs updating

Reply:
Fixed now.